src/components/GapList/index.jsx

```diff
@@ -11,12 +11,12 @@ const GapList = ({ items, xGap = 'm', yGap = 'm', valign = 'top' }) => (
     {items.map((item, i) => (
       <div
         key={i}
-        className={cx(
+        className={cc([
           'display--inlineBlock',
           `valign--${valign}`,
           `margin--x--${xGap}`,
-          `margin--y--${yGap}`
-        )}
+          `margin--y--${yGap}`,
```

Comment:
jscodeshift does a good job with code style - I like that it added trailing commas

Reply:
This is most likely Prettier potentially. JSCodeshift does do a very good job on formatting, but it's not as smart as Prettier. One of the bad parts of not having prettier in cbi-site yet, is running the transforms on cbi-site introduces formatting errors because jscodeshift can't know the opinionated formatting rules of eslint.